influxdb/client.py

```diff
@@ -219,7 +235,10 @@ def request(self, url, method='GET', params=None, data=None,
         :raises InfluxDBClientError: if the response code is not the
             same as `expected_response_code` and is not a server error code
         """
-        url = "{0}/{1}".format(self._baseurl, url)
+        if self.__uri == '':
+            url = "{0}/{1}".format(self._baseurl, url)
+        else:
+            url = "{0}/{1}/{2}".format(self._baseurl, self.__uri, url)
```

Comment:
Why are you doing this here rather than at `self.__baseurl` initialization?
Also, the condition could be removed if `format` was replaced with `urlparse.urljoin`.